ui/src/JobDetails.js

```diff
@@ -255,7 +256,10 @@ class JobDetails extends Component {
             {jobDetailsFragment}
             {buttonFragment}
             {this.props.job &&
-              this.props.authorisedActivities.includes("LOAD_SAMPLE") &&
+              this.props.job.jobType === this.props.loadJobType &&
```

Comment:
I think you can safely leave job type out of it and just check the permission... assume that the user of the component knows what they're doing